wdkt/src/main/kotlin/net/nprod/lotus/chemistry/Conversion.kt

```diff
@@ -37,3 +38,13 @@ fun smilesToCanonical(smiles: String): String {
     val atomContainer = SmilesParser(SilentChemObjectBuilder.getInstance()).parseSmiles(smiles)
     return canonicalGenerator.create(atomContainer)
 }
+
+fun smilesToMass(smiles: String): String {
+    val atomContainer = SmilesParser(SilentChemObjectBuilder.getInstance()).parseSmiles(smiles)
+    val mass = AtomContainerManipulator.getMass(atomContainer)
+    val sb = StringBuilder()
+    sb.append("+")
```

Comment:
Why a plus and U483261? is that because it is a special datatype?

Reply:
You can probably not set it as string if you need the unit you will have to make it send as the special type.

Reply:
I just checked https://www.wikidata.org/wiki/Property:P2067 and indeed it is a "quantity" data type...

Reply:
Anyway have to build a reference to the statement with https://www.wikidata.org/wiki/Property:P887 + https://www.wikidata.org/wiki/Q113907573